### ISSUE TITLE
Reactivate Ticking sound at least as a configurable option

### ISSUE THREAD

**Issue by basxib** [tomighty/tomighty-windows#14]
I've just installed the latest beta version and found out that the ticking sound was removed, even though most users don't like it, I do and it helps me focus better.

**Comment by BorisDNK** [comment]
I agree with you. I use tomighty because it has ticking sound. It make focus on a work with ticking sound. 
Please, add options.

**Comment by StephenHosking** [comment]
I switched from Tomighty to the commercial PomoDoneApp, and I've been very happy with it.

Pomodone starts each Pom with a loud "tick...tick..tick..", which quickly fades to nothing after eight ticks. I find it very good for focusing. There isn't an option for continues ticking, however. It's the eight, or nothing.
